zkevm-circuits/src/copy_circuit/test.rs

```diff
@@ -468,6 +476,9 @@ fn copy_circuit_invalid_codecopy() {
 
     assert_error_matches(
         test_copy_circuit_from_block(block),
+        #[cfg(feature = "dual_bytecode")]
+        vec!["Bytecode lookup", "rw lookup"],
+        #[cfg(not(feature = "dual_bytecode"))]
```

Comment:
wired... nice if we could unify them..

Reply:
the same feeling, the error messages is the same but the order changes. will find out the root cause after all test passing.

Reply:
after fixed copy lookup, the error becomes the same now as before.